packages/block-editor/src/components/block-selection-clearer/index.js

```diff
@@ -36,13 +41,13 @@ export function useBlockSelectionClearer() {
 				clearSelectedBlock();
 			}
 
-			node.addEventListener( 'mousedown', onMouseDown );
+			node.addEventListener( 'click', onMouseDown );
 
 			return () => {
-				node.removeEventListener( 'mousedown', onMouseDown );
+				node.removeEventListener( 'click', onMouseDown );
```

Comment:
What's the reason behind changing the event types?

Reply:
This was an errant commit based on a previous attempt at fixing this.  This has been removed.